pkg/gameserversets/controller.go

```diff
@@ -606,5 +606,21 @@ func computeStatus(list []*agonesv1.GameServer) agonesv1.GameServerSetStatus {
 		}
 	}
 
+	if runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		// to make this code simpler, we will loop around the gs list twice.
+		status.Players = &agonesv1.AggregatedPlayerStatus{}
+		for _, gs := range list {
+			if gs.ObjectMeta.DeletionTimestamp.IsZero() &&
+				(gs.Status.State == agonesv1.GameServerStateReady ||
```

Comment:
If a GS is ready or reserved, does it still have any player?

Reply:
Good question :thinking: 

In theory it _might_, say leftover from a previous session, and the GameServer has gone back into a pool, and the players are still connected.

I figured it made sense to err on the side of more flexible.